scripts/prerelease.mjs

```diff
@@ -0,0 +1,19 @@
+import moment from 'moment';
+import { setVersion } from './utils/pre-release.mjs';
+import { spawnProcess } from './utils/processes.mjs';
+
+const commitSha = (await spawnProcess('git rev-parse HEAD', { silent: true }))
+  .stdout
+  .toString()
+  .slice(0, 7);
+const currentDate = moment().format('YYYYMMDD');
+const currentBranchName = (await spawnProcess('git rev-parse --abbrev-ref HEAD', { silent: true })).stdout;
+let versionNumber = '0.0.0';
+
+if (currentBranchName.startsWith('release/')) {
+  versionNumber = currentBranchName.split('/')[1];
+}
```

Comment:
This will determine the version number as `[version]-docs` in case of running on `release/[version]-docs` branch, resulting in publishing a package versioned `[version]-docs-next-[commitSha]-[currentDate]`.

I think we can either rethink the release docs branch naming, or maybe get the package version number from `package.json`, which should already be correct on the release branch.

Reply:
The best fix would be reading the current Handsontable version from `package.json` file (the one in the root directory). I would propose using the same import as [here](https://github.com/handsontable/handsontable/blob/develop/scripts/utils/filesystem.mjs#L2), for example.

Reply:
@maciejbo-handsontable can I ask you to update the PR and add the `import` that I mentioned above?

Reply:
@budnix @jansiegel Hi! Requested change has been implemented (but instead of `package.json`, I'm reading version from  `hot.config.js` file).